### PR TITLE
[Federation] Fix the comments on FederationNameAnnotation

### PR DESCRIPTION
**Pull request by perotinus**
Follow-up for minor issues raised in #42683

```release-note
NONE
```

### CODE REVIEW

File: federation/apis/federation/annotations.go [hunk 1]
```diff
@@ -17,8 +17,9 @@ limitations under the License.
 package federation
 
 // FederationNameAnnotation is the annotation which holds the name of
-// the federation that an object is associated with. It must be
-// applied to all API objects associated with that federation.
+// the federation that that a federation control plane component is associated
+// with. It must be applied to all the API types that represent that federation
+// control plane components in the host cluster.
```
Comment (perotinus) [inline federation/apis/federation/annotations.go]: @madhusudancs Currently, I've been annotating objects in both the host and joining clusters. Do we just want to annotate objects in the joining cluster?

**Comment by perotinus** [comment]
cc @madhusudancs

**Comment by madhusudancs** [comment]
A minor nit.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[federation/apis/federation/annotations.go, line 20 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-KhdvZ3h-jSOxVgannqf:-KhdvZ3i79Ab5AT6YUot:b-7l2i0) ([raw file](https://github.com/kubernetes/kubernetes/blob/315b38e590f36379132ea2a07f88ae4e9a7b76ff/federation/apis/federation/annotations.go#L20)):*
> ```Go
> 
> // FederationNameAnnotation is the annotation which holds the name of
> // the federation that that a federation control plane component is associated
> ```

s/that that/that/

---

*[federation/apis/federation/annotations.go, line 22 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-KhdOpI5iu0N-p8JGOgT-r1-22:-Khdvh2mhyIq0emj-d1T:b-7oueif) ([raw file](https://github.com/kubernetes/kubernetes/blob/315b38e590f36379132ea2a07f88ae4e9a7b76ff/federation/apis/federation/annotations.go#L22)):*
<details><summary><i>Previously, perotinus (Jonathan MacMillan) wrote…</i></summary><blockquote>

@madhusudancs Currently, I've been annotating objects in both the host and joining clusters. Do we just want to annotate objects in the joining cluster?
</blockquote></details>

We definitely want to annotate objects in the host cluster. We create a number of objects in the host cluster and we want to keep track of them.

Annotating objects in the joining cluster is useful too, esp. if the cluster is part of multiple federations. Feel free to fix the doc string accordingly.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-:-Khdw36onn7pCEa9HD6i:b9rmf7o)*

**Comment by perotinus** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[federation/apis/federation/annotations.go, line 20 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-KhdvZ3h-jSOxVgannqf:-Khe7Zk7jgx491ZT1bQ3:b40dv2l) ([raw file](https://github.com/kubernetes/kubernetes/blob/315b38e590f36379132ea2a07f88ae4e9a7b76ff/federation/apis/federation/annotations.go#L20)):*
<details><summary><i>Previously, madhusudancs (Madhusudan.C.S) wrote…</i></summary><blockquote>

s/that that/that/
</blockquote></details>

D'oh. Fixed.

---

*[federation/apis/federation/annotations.go, line 22 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-KhdOpI5iu0N-p8JGOgT-r1-22:-Khe3d6o8SJ3zL3HY5vc:b-qesh72) ([raw file](https://github.com/kubernetes/kubernetes/blob/315b38e590f36379132ea2a07f88ae4e9a7b76ff/federation/apis/federation/annotations.go#L22)):*
<details><summary><i>Previously, madhusudancs (Madhusudan.C.S) wrote…</i></summary><blockquote>

We definitely want to annotate objects in the host cluster. We create a number of objects in the host cluster and we want to keep track of them.

Annotating objects in the joining cluster is useful too, esp. if the cluster is part of multiple federations. Feel free to fix the doc string accordingly.
</blockquote></details>

Oh, yes, of course! I think that's what I meant to say.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472)*

**Comment by nikhiljindal** [comment]
@k8s-bot ok to test

**Comment by madhusudancs** [comment]
/lgtm
/approve

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/44472#-:-KiaXVhUHCghvsPBQi1W:b6yv7yi)*

**Pull request merged**